### PR TITLE
Small fix

### PR DESCRIPTION
**Pull request by ovaistariq**

### CODE REVIEW

File: docs/concepts/mongodb-compatibility/index.mdx [hunk 1]
```diff
@@ -8,8 +8,9 @@ connection string to connect to Tigris instead of MongoDB Atlas. Then, update
 your application functionality to take advantage of additional Tigris
 functionality according to your roadmap.
 
-**The first beta release of Tigris MongoDB compatibility offers MongoDB 6.0+
-basic CRUD functionality**. You can communicate with the Tigris using any open source
+**The first beta release of Tigris MongoDB compatibility offers support for MongoDB 6.0+
+[CRUD and Aggregation operations](docs/concepts/mongodb-compatibility/supported-features)**.
```
Comment (leggetter) [inline docs/concepts/mongodb-compatibility/index.mdx:12]:
```suggestion
[CRUD and Aggregation operations](/docs/concepts/mongodb-compatibility/supported-features/)**.
```

Comment (leggetter) [reply docs/concepts/mongodb-compatibility/index.mdx:12]: The missing `/` at the start probably caused the broken link.

Comment (ovaistariq) [reply docs/concepts/mongodb-compatibility/index.mdx]: Fixed

File: docs/concepts/mongodb-compatibility/index.mdx [hunk 2]
```diff
@@ -8,8 +8,9 @@ connection string to connect to Tigris instead of MongoDB Atlas. Then, update
 your application functionality to take advantage of additional Tigris
 functionality according to your roadmap.
 
-**The first beta release of Tigris MongoDB compatibility offers MongoDB 6.0+
-basic CRUD functionality**. You can communicate with the Tigris using any open source
+**The first beta release of Tigris MongoDB compatibility offers support for MongoDB 6.0+
+[CRUD and Aggregation operations](/docs/concepts/mongodb-compatibility/supported-features)**.
```
Comment (leggetter) [inline docs/concepts/mongodb-compatibility/index.mdx:12]:
```suggestion
[CRUD and Aggregation operations](/docs/concepts/mongodb-compatibility/supported-features/)**.
```

Comment (leggetter) [reply docs/concepts/mongodb-compatibility/index.mdx:12]: The `/` at the end stops a 307 redirect.

**Review by leggetter — APPROVED** [review]

**Pull request merged**